app/routes/bank.tsx

```diff
@@ -0,0 +1,28 @@
+import { NavLink, Outlet } from "@remix-run/react"
+import type { LoaderArgs } from "@remix-run/server-runtime"
+import { PageHeader } from "~/components/elements/PageHeader"
+import { redirectIfUserNotLoggedIn } from "~/services/auth/session.server"
+
+export const loader = async ({ request }: LoaderArgs) => {
+  await redirectIfUserNotLoggedIn(request)
+
+  return {}
+}
+
+export default function BankIndex() {
+  return (
+    <div className="p-12">
+      <PageHeader>Bank of Canadian English</PageHeader>
+      <nav className="bg-slate-100">
+        <h2 className="my-8 text-xl font-bold">
+          Links: <NavLink to="/bank/stats">Statistics</NavLink> •{" "}
+          <NavLink to="/bank/create">New</NavLink> •{" "}
+          <NavLink to="/bank/own">Show own</NavLink>
```

Comment:
Right now this is throwing an error for me-- I think it's cause I'm not in my local database of users, but the error is uncaught so I don't know if we want to change that or not.

I'll add my user to my local database and see how that changes things

Reply:
update: it does fix it

Reply:
Good catch. I'll create a separate issue for this.